run_tests.sh

```diff
@@ -1,9 +1,14 @@
 set -e
 docker run --name test_mongo -d -p 8545:8545 -p 8080:8080 -p 27017:27017 mongo
+trap "docker stop test_mongo && docker rm test_mongo" EXIT SIGINT SIGTERM
+# TODO restore go test -tags=integration ./...
+go test ./...
```

Comment:
I will open an issue to fix the go integration tests. None of the go tests were running before, so this is still an incremental improvement for now.

Reply:
I moved this step in to the docker build (behind a `build-arg` switch) so we are building/caching in the same place.

Reply:
The integration tests should be able to call out to the existing mongo container from there (we already had it running anyways), but they are still off for now.